src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1352,6 +1303,40 @@ namespace WaterToAirHeatPumpSimple {
 			if ( RatedCapCoolSensDes < SmallLoad ) {
 				RatedCapCoolSensDes = 0.0;
 			}
+			if ( RatedCapCoolTotalAutoSized && RatedCapCoolSensAutoSized ) {
+				if ( RatedCapCoolSensDes > RatedCapCoolTotalDes ) {
```

Comment:
Should the autosized test be changed to "or" so if either is autosized this test is performed?